internal/decodejpeg/decode_windows.go

```diff
@@ -0,0 +1,36 @@
+package decodejpeg
```

Comment:
Is this code duplication necessary due to some cgo-magic? No way around it using `#IFDEF`s or similar in either `decode.go` or `decode.h`?

Reply:
Go uses convention over configuration - your linux go compiler will just ignore _windows files.

Reply:
It's still a lot of unnecessary duplication. It feels like this should be solvable in a better (_i.e._ more maintainable) way. In a c program, I would use _e.g._ `#ifdef _WIN64`, but for linking those directives would probably need to go in the makefile, in which case it might be tricky (and perhaps not go-like).

Bottom line: I don't like this, but I guess it might be ok for now. If we make a separate package of this, however, I think it should be cleaned up.

Reply:
OK, found a cleaner way to make it work.